language/de.json

```diff
@@ -2,7 +2,7 @@
   "semantics": [
     {
       "label": "Beschreibung für das Eingabefeld",
-      "description": "Beschreibung, was der Nutzer eingeben soll"
```

Comment:
Please be careful with replacing "Nutzer" or "Nutzerinnen" or "NutzerInnen" (user/user) by "Lernende" (students). H5P is mainly used in educational scenarios, but especially this generic text input field can be used in many other scenarios as well.